src/main/kotlin/com/strumenta/kolasu/model/Navigation.kt

```diff
@@ -3,6 +3,7 @@ package com.strumenta.kolasu.model
 /**
  * @return the nearest ancestor of this node that is an instance of klass.
  */
+@Suppress("UNCHECKED_CAST")
```

Comment:
Should we add a comment to explain why this is ok here?

Reply:
This one is gone when we merge the node_sequences branch.